patterns/hidden-404.php

```diff
@@ -6,9 +6,9 @@
  */
 ?>
 <!-- wp:heading {"className":"has-text-align-center"} -->
-<h2 class="has-text-align-center"><?php esc_html( _x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ) ); ?></h2>
+<h2 class="has-text-align-center"><?php echo esc_html( _x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ) ); ?></h2>
```

Comment:
What is the reason between using `echo esc_html` here and `esc_html_e` below?

Reply:
Hmm actually it should be echo esc_html_x():  [esc_html_x](https://developer.wordpress.org/reference/functions/esc_html_x/) does not have variant that both echoes, escapes, translates, and uses the context